spec/datadog/tracing/sampling/span/rule_parser_spec.rb

```diff
@@ -0,0 +1,154 @@
+require 'datadog/tracing/sampling/span/rule_parser'
+
+RSpec.describe Datadog::Tracing::Sampling::Span::RuleParser do
+  describe '.parse_json' do
+    subject(:parse) { described_class.parse_json(rules_string) }
+    let(:rules_string) { JSON.dump(json_input) }
+    let(:json_input) { [] }
+
+    shared_examples 'does not modify span' do
+      it { expect { sample! }.to_not(change { span_op.send(:build_span).to_hash }) }
+    end
```

Comment:
This looks like a copy-pasta accident, it's not used anywhere and not relevant to the rule parser :)

```suggestion
```